BraveRewardsUI/Tipping/TippingViewController.swift

```diff
@@ -63,6 +72,73 @@ class TippingViewController: UIViewController, UIViewControllerTransitioningDele
     tippingView.optionSelectionView.options = [1.0, 5.0, 10.0].map {
       TippingOption.batAmount(BATValue($0), dollarValue: state.ledger.dollarStringForBATAmount($0) ?? "")
     }
+    
+    loadPublisherBanner()
+  }
+  
+  // MARK: - Networking
+  private func loadPublisherBanner() {
+    state.ledger.publisherBanner(forId: self.publisherInfo.id) { [weak self] banner in
+      guard let self = self, let banner = banner else { return }
+      
+      self.tippingView.overviewView.titleLabel.text = banner.title.isEmpty ? Strings.TippingOverviewTitle : banner.title
+      self.tippingView.overviewView.bodyLabel.text = banner.desc.isEmpty ? Strings.TippingOverviewBody : banner.desc
+      
+      self.downloadImage(url: banner.background, { image in
+        self.tippingView.overviewView.headerView.image = image
+      })
+      
+      self.downloadImage(url: self.publisherInfo.faviconUrl, { image in
+        self.tippingView.overviewView.faviconImageView.image = image
+      })
+      
+      self.tippingView.overviewView.socialStackView.arrangedSubviews.forEach({ $0.removeFromSuperview() })
+      
+      if !banner.social.isEmpty {
+        banner.social.keys.forEach({
+          //When testing with "3zsistemi.si", the values come back empty, but we still have valid keys..
+          //It seems we don't care about the values as long as the key exists, that user has this media..
+          if let mediaType = BravePublisherSocialMedia(rawValue: $0) {
+            let icons: [BravePublisherSocialMedia: String] = [.youtube: "pub-youtube",
+                                                              .twitter: "pub-twitter",
+                                                              .twitch: "pub-twitch"]
+            
+            if let icon = icons[mediaType] {
+              self.tippingView.overviewView.socialStackView.addArrangedSubview(UIImageView(image: UIImage(frameworkResourceNamed: icon)))
+            }
+          }
+        })
+      } else {
+        ["pub-youtube", "pub-twitter", "pub-twitch"].forEach({
```

Comment:
If its empty I don't think we're supposed to show any of these

Reply:
Fixed.